.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
       os: linux
 
 before_install:
-  - echo -e "{\n  \"ipv6\":true,\n  \"fixed-cidr-v6\":\"2001:db8:1::/64\"\n}" | sudo tee /etc/docker/daemon.json
+  - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
```

Comment:
Spaces after `:` and `,` will improve readability.

Reply:
Adding a space after the colons (`:`) will break Travis CI's linter.

Reply:
I don't use the linter, will it break the build?

Its the tiniest of nitpicks either way...

Reply:
hmmm (reference: https://github.com/travis-ci/travis-ci/issues/7658#issuecomment-297098649)

Reply:
Using:
```cr
before_install:
  - >
      echo '{"ipv6": true, "fixed-cidr-v6": "2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
  - sudo service docker restart
  - bin/ci prepare_system
```
Would work :smile: